Readme.md

```diff
@@ -0,0 +1,24 @@
+# How to use the split utility
+
+The split utility is used by giving it a LLVM bitcode file and it splits all of
+the functions in it into separate files. 
+
+Obtaining the final LLVM bitcode file can be dune in multiple ways. The easiest
+one being to use
+[whole-program-llvm](https://github.com/travitch/whole-program-llvm) that gives
+you the bitcode used to create the executable.
+
+For example to get the bitcode from the cpython executable you build it like
+that: 
+```
+LLVM_COMPILER="clang" CC="wllvm" ./configure 
+LLVM_COMPILER="clang" make
+``` 
+Then extract the bitcode file from the binary with:
+```
+extract-bc python 
+``` 
+The last step would be to execute the split utility. This can be done
```

Comment:
Missing newline before here?

Reply:
I used the vim formatting so I think this is fine.

Reply:
This is a new paragraph, so while vim might be happy with no newline, humans like me aren't so happy ;)

Reply:
I think we just need to fix this then we're done.

Reply:
sry I missed that. New lines here: 73fc5ec